hkmc2/shared/src/test/mlscript/bbml/bbBasics.mls

```diff
@@ -20,7 +20,7 @@ false
 
 fun id(x) = x
 id
-//│ Type: forall outer, 'x: 'x -> 'x
+//│ Type: [outer, 'x] -> 'x -> 'x
```

Comment:
Can't the `outer` type be ignored here?

Reply:
Good suggestion. I also fixed the problem with simplification of higher-ranked polymorphic function types.